app/models/region_trail_link.rb

```diff
@@ -1,4 +0,0 @@
-class RegionTrailLink < ActiveRecord::Base
```

Comment:
@cchchoi1986 Take a look at this. Trail and Region have one-to-many relationship, not many-to-many, so we changed it.